README.md

```diff
@@ -1,46 +1,109 @@
+- Click [HERE](https://art-from-the-heart-mr-ryan12.vercel.app/) to view the deployed site
 
-## Available Scripts
+## Table of Contents
+- [Abstract](#abstract)
+- [Tech Used](#tech-used)
+- [Installation and Set-Up](#installation-and-set-up)
+- [Features](#features)
+- [See the Site in Action](#see-the-site-in-action)
+- [Future Goals](#future-goals)
+- [Created By](#created-by)
+- [Credit](#credit)
 
-In the project directory, you can run:
+## Abstract
 
-### `npm start`
+*Tired of pretending to like ugly art in museums? We are too.*
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+Art From The Heart is a web application that allows users to overcome internal and external biases to answer the only question that matters: Do I like this work of art? 
 
-The page will reload if you make edits.\
-You will also see any lint errors in the console.
+The homepage greets a user with categories of art pulled directly from the Art Institute of Chicago. Upon clicking a category, a random work of art will be displayed for a user to rate up or down. Only after they rate the work of art will the details about the author, creation date, description of the work, etc. be shown to the user. This allows people to view and enjoy art without the implications of certain biases being forced upon them such as ___ ___ or ____. After they have rated the art they can then choose to see another art piece from that category or they can choose to go home and select another category. 
 
-### `npm test`
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+## Tech Used
 
-### `npm run build`
+- React 
+- JSX
+- TypeScript
+- Cypress Testing
+- HTML5
+- SASS/CSS
+- JavaScript
+- API Fetch Calls
+- React Router
+- Framer Motion
+- Art Data API used for Network Requests (https://api.artic.edu/docs/)
+- Project Structure Organized with [GitHub Project Board](https://github.com/mr-ryan12/art-from-the-heart/projects/2)
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+## Installation and Set-Up
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+To install this project, please see below:
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+```bash
+1. Clone down this Repository using `git clone`
+2. Next, `cd` into the cloned repository and `npm install` to install library dependencies
+3. Finally, run `npm start` and open up localhost at the port indicated to view the webpage (http://localhost:3000/)
+```
+Alternatively, you can access the deployed site [HERE](https://art-from-the-heart-mr-ryan12.vercel.app/)
+    
+## Features
```

Comment:
Maybe we could add the full terminal command "git clone git@github.com:mr-ryan12/art-from-the-heart.git" and an instruction to close the local server "CTRL + c (mac)"??? Glad the npm install works!